src/controllers/ExperimentViewController.ts

```diff
@@ -109,6 +113,14 @@ export class ExperimentViewController {
     }
   }
 
+  /**
+   * Generates a random interval time between two bounds set in state.
+   */
+  generateRandomInterval(): number {
```

Comment:
I'm not sure why this has to be a method on the ExperimentViewController. It's only used in one place, and is only called once. Might be better to _not_ have a method, or at least put this in `FearConditioningModuleViewController`.

Reply:
Yeah it's a weird one, it was put here because the interval bounds are part of the experiment definition and that kind of delegates their management to the experiment instead of the module itself.

I'll move it to the module level and if we need it again I'll move it back up. I still think it should be a method as we want the code base to be as readable as possible.

Reply:
I don't think it's a good idea to have this as a method because:

1. It's only used once, which makes the method unnecessary
2. It's only a couple of lines long
3. Every new method on a class comes with some maintenance burden, and I don't think it's worth it in this case

I also don't agree it would make the code more readable. I could argue that it would make it _less_ readable because it would require future devs to find the method definition when reading `FearConditioningModuleViewController.render`.

On the scale of simple vs complex, adding a new method for this specific functionality falls on the complex side when compared to just writing out the code.

However, if you feel strongly that this should be a method, we can keep it as is.

Reply:
Yeah fair point, I've switched that switched out.